src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
```

Comment:
:information_source: Using @author tag is not recommended as it can be automatically generated by the version control system. (Severity: Minor)